### DISCUSSION TITLE
Allow Prefixes for Connection Endpoints to Indicate Model Sources

### DISCUSSION THREAD

**Discussion by Manuelbaun** [open-webui/open-webui#6181] [Ideas]
When adding a new connection endpoint, I would like the ability to include a prefix that indicates the source of the models in my list. For instance, I use both local Ollama and Groq, and both provide similar Llama models. When I select a model, I can’t easily determine whether it originates from my local Ollama or from Groq.

It would be beneficial to have the option to add a prefix to the OpenAPI endpoint or any other endpoint, allowing users to clearly identify the source of each model.

**Comment by mwwhited** [comment]
This would be handing to be able to group and filter by this as well.  Maybe having a user friendly name of for the endpoint.  This will be more important in mixed model scenarios.  I could see cases where maybe some work is okay to be slower and more secure so it has to be ran on a local service while something more generic and less secure could sent out of a public shared service like OpenAI.
